packages/client/components/contacts/ContactsForm.tsx

```diff
@@ -131,6 +131,21 @@ export default function ContactsForm(props: Props) {
         }
     }
 
+    function handleDeleteContact(): void {
+        Alert.alert("Do you want to delete this contact?", "You cannot undo this action.", [
+            { text: "Cancel", style: "cancel" },
+            {
+                text: "Delete",
+                style: "destructive",
+                onPress: async () => {
+                    await contactsGateway.deleteContact({ id: user.id, name: props.contact?.contactName }, token);
+                    props.navigation.goBack();
+                    props.navigation.goBack();
```

Comment:
I'd wrap this in an if statement so there is no chance of sending a null value.

```suggestion
                    if (props.contact) {
                        await contactsGateway.deleteContact({ id: user.id, name: props.contact.contactName }, token);
                        props.navigation.goBack();
                        props.navigation.goBack();
                    }
```